Vagrantfile

```diff
@@ -0,0 +1,135 @@
+# -*- mode: ruby -*-
```

Comment:
Refer to some existing JIRA or else create one explaining how we might be able to economise on these cut and pasted vagrant files